rust/windows-client/src-tauri/Cargo.toml

```diff
@@ -46,6 +46,7 @@ rand = "0.8.5"
 windows-implement = "0.52.0"
 sadness-generator = "0.5.0"
 bincode = "1.3.3"
+native-dialog = "0.7.0"
```

Comment:
Do we necessarily need to use a cross-platform library here? It appears that it comes with a few limitations: https://github.com/native-dialog-rs/native-dialog-rs?tab=readme-ov-file#misc

Does Tauri not have a way of showing a dialog?

Reply:
Tauri does, but I got this stuck in my head cause I was working on #3464 at the same time and I was going to use `native-dialog` for that one anyway. (And in that one, Tauri isn't available)

I agree it's overkill, we could call Win32's `MessageBox` API directly if we need to later